### DISCUSSION TITLE
Notification URL para Suscripciones

### DISCUSSION THREAD

**Discussion by fertandil87** [mercadopago/sdk-js#143] [General]
Hola estoy automatizando todo el proceso de suscripciones de mi producto, y me estoy topando con algo...
normalmente en algunos procesamientos de pagos yo puedo setear el notificacion_url (que es muy practico porque tengo muchos servidores en distintos dominios y distantas bases de datos)
Pero para a la hora de crear una suscripcion, primero obtengo el card token id usando 

const mp = new MercadoPago('APP_USR-e8cxxxxxxxxxx');
const cardForm = mp.cardForm({ y luego cardForm.getCardFormData()

hasta ahi todo bien tengo el token y llamo al endpoint
POST https://api.mercadopago.com/preapproval
donde le paso:
{
  "preapproval_plan_id": "2c93808xxxxxxxxxxxx",
  "reason": "Yoga classes suscription",
  "external_reference": "YG-1234",
  "back_url": "https://www.xxxxx.com.ar",
  "status": "authorized",
  "card_token_id": "947a4c2402xxxxxxxxxxxxx",
  "payer_email": "xxxxxxxxxx@gmail.com",
  "notification_url": "https://webh.com/asdasdfsadfsdafasdf"
}

el problema es que nunca llegan las notificaciones ahi, pero si me llegan 2 tipos de notificaciones distintas al webhook de mi cuenta principal y al webhook de mi app, pero no al de la notification_url, y crear una app para cada servidor no seria lo mas conveniente.

Que podria hacer en este caso?

**Comment by romerosilva-meli** [comment]
Hola @fertandil87 como estás?

Este espacio de discussion es solo para preguntas relacionadas con SDK Js (frontend). Le pediré que [abra un ticket de soporte](https://www.mercadopago.com.ar/developers/es/support/center/tickets) para abordar su problema.
